docs/_quartodoc-core.yml

```diff
@@ -98,6 +98,23 @@ quartodoc:
         - ui.input_file
         - ui.download_button
         - ui.download_link
+    - title: Bookmarking
+      desc: Saving and restoring app state
+      contents:
+        - ui.input_bookmark_button
+        - bookmark.restore_input
+        - bookmark.Bookmark
+        - bookmark.BookmarkState
+        - bookmark.RestoreState
+        - kind: page
+          path: bookmark_integration
+          summary:
+            name: "Integration"
+            desc: "Decorators to set save and restore directories."
+          flatten: true
+          contents:
+            - bookmark.set_save_dir
+            - bookmark.set_restore_dir
```

Comment:
Have the ability to set these at the `App` level.

Reply:
[`a50860e` (#1870)](https://github.com/posit-dev/py-shiny/pull/1870/commits/a50860e21c5a86af723e757f04829fa4457e2549)